stack/base/before-notebook.d/20_setup-ssh.sh

```diff
@@ -11,5 +11,11 @@ if [[ ! -f  /home/${NB_USER}/.ssh/id_rsa ]]; then
   ssh-keygen -f /home/${NB_USER}/.ssh/id_rsa -t rsa -b 4096 -m PEM -N ''
 fi
 
+# Set by source load-singlesshagent.sh script
+echo $'\n# Load singlesshagent on shell startup.
```

Comment:
@unkcpz I've noticen an issue with this code, the '.bashrc' gets appended everytime the container is started, so this code gets needlessly duplicated. I would proposed to move this in the `10_prepare-home-rc.sh`, where you can append it just one when the `.bashrc` is first copied over.

Reply:
I think you are correct. But I initially didn't move this to `10_prepare-home-rc.sh` in order to keep all ssh related setting in one file and to make sure the command runs after running the ssh-keygen (which is not necessary but for readability).
Maybe instead of downloading the `load-singlesshagent.sh` on the fly, it is easy to manually maintain it in this repo and add a file existence check before running it? What you think?

Reply:
@unkcpz I've created #357 to keep track of this issue. I assigned you and put to next sprint since I think it would be good to fix before the next docker stack release. Feel free to disagree though...